### ISSUE TITLE
Retrieve entire Dataset/metadata from remote PACS?

### ISSUE THREAD

**Issue by barchard** [patmun/pynetdicom#54]
Hi, it seems as though you have to specify each key you want returned from the Remote PACS. Is there a way to say "give me all of the meta data that match this query"? For example, I want to find all Patients and their respective metadata that look like `*john*`.

Thanks

**Comment by naggie** [comment]
No, you have to specify each key you want to seach in as per the DICOM protocol.
